src/schemas/json/smapi-manifest-3.17.0.json

```
@@ -0,0 +1,89 @@
+{
+  "$comment": "https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest",
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "definitions": {
+    "semver": {
+      "type": "string",
+      "description": "Semantic version number (e.g. 1.2.3-alpha.4).",
+      "pattern": "^(\\d|[1-9]\\d*)\\.(\\d|[1-9]\\d*)\\.(\\d|[1-9]\\d*)(-(alpha|beta|rc)(\\.(\\d|[1-9]\\d*))?)?$"
+    },
+    "slug": {
+      "type": "string",
+      "description": "Slug type (alphanumeric and periods).",
+      "pattern": "^[\\.\\w\\d]+$"
+    },
+    "dependency": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "UniqueID": {
+          "description": "The unique ID of the mod this content pack is for.",
+          "$ref": "#/definitions/slug"
+        },
+        "MinimumVersion": {
+          "description": "The minimum version of the mod this content pack is for. If the player doesn't have the minimum version, SMAPI will show an error message.",
+          "$ref": "#/definitions/semver"
+        }
+      }
+    },
+    "dllFile": {
+      "type": "string",
+      "description": "Relative path to a DLL file.",
+      "pattern": "^.+\\.dll$"
+    }
+  },
+  "description": "Manifest for a SMAPI mod.",
+  "properties": {
+    "Name": {
+      "type": "string",
+      "description": "The mod name. SMAPI uses this in player messages, logs, and errors.",
```

```suggestion
      "description": "The current mod name\nhttps://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest#Fields,
```

Optional fix: it's better keep descriptions concise but add link to the docs.